pkg/node/node.go

```diff
@@ -171,6 +172,14 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 			return nil, err
 		}
 
+		chainID, err = swapBackend.ChainID(p2pCtx)
+		if err != nil {
+			logger.Infof("could not connect to backend at %v. A working blockchain node (for goerli network in production) is required. Check your node or specify another node using --swap-endpoint.", o.SwapEndpoint)
```

Comment:
not sure about using `Info` here. maybe `Warn` or even `Error`? also, the message says `"could not connect to backend"` but could it not be something else?

Reply:
not sure why this was not `Error` before, fixed now. There should not be any other error (if there were it would be printed anyway due to the returned error in the next line). The reason why it explicitly mentions the connection here is because bee will otherwise just terminate with `Error: could not get chain id from ethereum backend: Post "http://localhost:8545": dial tcp [::1]:8545: connect: connection refused` which could be confusing to users.